### ISSUE TITLE
AttributeError missing information

### ISSUE THREAD

**Issue by fatherofinvention** [aroberge/friendly-traceback#43]
Hello, 

As requested, I am reporting an issue in which an exception was encountered but no information was known. This was in version 0.0.28 running in Thonny 3.2.7.

**Code to repro:**
```
class Vehicle:
    '''Vehicle parent class with class and instance variables'''
    wheels = 4

    def __init__(self, make, model):
        self.make = make
        self.model = model

class Car(Vehicle):
    '''Car child class'''
    
    def __init__(self, color, mileage):
        self.color = color
        self.mileage = mileage

my_car = Car('Green', 300)
my_car.model
```

**Error:**

> Python exception:
>         AttributeError: 'Car' object has no attribute 'model'
>         
>     No information is known about this exception.
>     Please report this example to
>     https://github.com/aroberge/friendly-traceback/issues
>     
>     Execution stopped on line 1 of file '<pyshell>'.
>     
> Problem: source of '<pyshell>' is not available

Thanks and keep up the good work!

**Comment by aroberge** [comment]
Thank you. I will try to add an explanation for such errors.

**Comment by aroberge** [comment]
Implemented in version 0.0.29.  Here's a simpler example than the one reported:

```
$ python -m friendly_traceback
```
```python
Friendly Console version 0.0.29. [Python version: 3.7.3]

>>> class Car:
...   def __init__(self, color):
...       self.color = color
...
>>> car = Car('green')
>>> car.model
```
```
    Python exception:
        AttributeError: 'Car' object has no attribute 'model'

    An attribute error occurs when the code contains something like
        object.x
    and 'x' is not a method or attribute (variable) belonging to 'object'

    Likely cause based on the information given by Python:
        In your program, the object is 'Car' and the attribute is 'model'.

    Execution stopped on line 1 of file '<friendly-console:3>'.

    -->1: car.model

    car: <Car object>
```
N.B. The location of the error will only be available if one uses the editor (not the repl) in Thonny or, as shown above, an REPL compatible with Friendly-traceback.
